common/web/sentry-manager/src/index.ts

```diff
@@ -109,7 +113,7 @@ namespace com.keyman {
         });
       }
 
-      if (event.request.url) {
+      if (event.request && event.request.url) {
```

Comment:
This wasn't causing errors, but I had to fix line 65 above so did this purely for my sanity at the same time.